layouts/partials/frontpage-announcement.html

```diff
@@ -1,14 +1,34 @@
-{{ if .Page.Param "announcement" }}
-<section lang="en" id="fp-announcement" style="background-color:{{ .Page.Param "announcement_bg" }}">
-  <aside >
-    <div class="content announcement main-section" data-nosnippet>
-
-      <h3>
-          {{ T "announcement_title" |  markdownify }}
-      </h3>
-      <p>{{ T "announcement_message" | markdownify }}</p>
-
-    </div>
-  </aside>
-</section>
+{{ $dateRegExp := "^[0-9]{4}-1[0-2]|0[1-9]-(?:3[01]|0[1-9]|[12][0-9])T(?:2[0-3]|[01][0-9]):(?:[0-5][0-9]):(?:60|[0-5][0-9])$" }}
+{{ $announcementShown := false }}
+{{ range $.Site.Data.announcements }}
+  {{ range .announcements }}
```

Comment:
@sftim, Since the two partials (front-page and announcement) are nearly identical, could you use one partial
template and provide the "id" string as an argument?

Reply:
[Yes](https://github.com/kubernetes/website/pull/25769#discussion_r547544855).

Reply:
Good. One announcement partial will be easier to maintain.
I am happy to see this move forward. Thanks.

Reply:
+1.

There was An Idea(tm) around having two shortcodes, but it didn't pan out. Feel free to remove it.

Reply:
Hi @sftim ,
I do not see a compelling reason to leave the frontpage-announcement partial in place.
I realize that it is easier to copy the changes to the existing partial so that the other layouts are not disrupted,
but these updates seem to be part of this PR. Are there other issues that prevent these changes?

These files should be updated, 
/layouts/_default/baseof.html
/layouts/_default/search.html

```
{{ block "announcement" . }}
      {{ if .IsHome }}
        {{ partial "frontpage-announcement.html" . }}
      {{ else }}
        {{ partial "announcement.html" . }}
      {{ end }}
    {{ end }}
```